libshaderc/include/shaderc.h

```diff
@@ -113,6 +113,17 @@ void shaderc_compile_options_release(shaderc_compile_options_t options);
 void shaderc_compile_options_add_macro_definition(
     shaderc_compile_options_t options, const char* name, const char* value);
 
+// Set the compiler to emit disassembly code.
```

Comment:
Like the other MR, should say that the byte array in the module returned from the compile method will consist of the SPIR-V assembly text.  And that preprocessing mode overrides this option, but this option overrides the default mode of generating a SPIR-V binary.